verification/VerifyBroadInternalRNAWithUMIs.wdl

```diff
@@ -0,0 +1,48 @@
+version 1.0
```

Comment:
Can we just use the VerifyRNAWithUMIs directly? Do we need this file?

Reply:
Yup, done